conda/recipes/mantid/bld.bat

```diff
@@ -16,9 +18,11 @@ cmake ^
     -DMANTID_FRAMEWORK_LIB=BUILD ^
     -DMANTID_QT_LIB=OFF ^
     -DENABLE_WORKBENCH=OFF ^
+    -GNinja ^
+    -DCMAKE_BUILD_TYPE=Release ^
     ..
 
 if errorlevel 1 exit 1
-cmake --build . --config Release
-cmake --build . --config Release --target install
+ninja
+ninja install
```

Comment:
Just want to check why this was changed? If `cmake` is configured with `-G Ninja` it will be using ninja to build with anyway

Reply:
It threw a "warning" about `--config Release` having no effect (ninja uses a compiler flag directly), so I just changed it to call `ninja` directly inline with the unix scripts.

Reply:
Cool, if that's inline with windows then we're froody